js/manipulate-card.js

```diff
@@ -22,15 +22,15 @@
   var closeCard = function () {
     var cardClose = map.querySelectorAll('.popup__close');
     for (var i = 0; i < window.utils.PIN_NUMBER; i++) {
-      cardClose[i].addEventListener('click', window.createElement.mapCardHidden);
+      cardClose[i].addEventListener('click', window.createElement.mapCardHide);
```

Comment:
```suggestion
      cardClose[i].addEventListener('click', window.createElement.mapCardHide);
```
неправильное название лисенера: https://up.htmlacademy.ru/javascript/18/criteries#d4

ты, кстати, какой вариант названий обработчиков событий используешь? Через on или через handler?

Reply:
через on